flexmeasures/data/models/audit_log.py

```diff
@@ -149,6 +149,10 @@ def add_record_for_attribute_update(
             affected_asset_id = asset_or_sensor.id
         event += f"attribute '{attribute_key}' to {attribute_value} from {old_value}"
 
+        # Truncate event string if necessary to prevent ERROR: value too long for type character varying(255)
```

Comment:
As we discussed, let's move this fix into its own PR. I looked at the AssetAPI patch code, and the audit log can also be saving on characters.
- No need to write asset name and ID (it's connected to the asset via a foreign key)
- "Attribute name:" can be "attr:" 
- "Old value:" can be "from:"
- "New value" can be "to:"
etc.

Then we can make sure not too much valuable information is cut off - which dimibishes the usefulness of the audit log.

Reply:
this was adressed in the pr.